ml-agents/mlagents/trainers/sac/network.py

```diff
@@ -121,10 +121,14 @@ def _create_cc_critic(self, hidden_value, scope, create_qs=True):
             self.h_size,
             self.join_scopes(scope, "value"),
         )
-
+        self.external_action_in = tf.placeholder(
```

Comment:
Does this change just shift the burden of who needs to have the placeholder?

Reply:
Yes, for SAC we can get away with moving the placeholder since it wasn't connected anywhere further down the graph.